bundles/paikkatietoikkuna/coordinatetransformation/resources/css/filesettings.css

```diff
@@ -24,17 +24,17 @@
     float: right;
     padding-left: 3em;
 }
-.file-select {
+.settingsSelect {
     float: left;
 }
-.file-select > select {
+.settingsSelect > select {
     float: right;
     width: 12em;
 }
 .cancel {
     clear: left;
     float: left;
 }
-.import {
+.done {
```

Comment:
At least .cancel and .done are so common words to be used as classes that this is bound to break something somewhere. Prefix all of these with coordinate-transform or similar.